JS Basics/Strings/Problem_3/index.js

```diff
@@ -0,0 +1,7 @@
+function countOccurences(str) {
+    return count = [...(str.match(/in/g))].length;;
+}
```

Comment:
Follow the task - the input should be an array, where the first element is the one we are looking for, and the second - the string in which we will be searching.